### PR TITLE
Readme restructuring

### PR DESCRIPTION
**Pull request by Sven-Seyfert**
Hi @Danp2 , hi @mlipok ,

I changed the previous presented ReadMe file to almost all recommendations of our discussion in issue #181 .
I also added a default CONTRIBUTING.md file to fit a bit more the "Community Standards" (like I hinted in the issue).

What I haven't done is move the FAQ section and also not stripping the content to a subset like you @Danp2 thought about the content of a FAQ.
- Why? Because I hope we can have a separate discussion (issue) about what should be in and what not.
- I hope this is fine for you both 🤞 ?

> My wish is to transform to the "new" ReadMe and maintain the content step by step (Keyword "Opera" as an example).

Best regards
Sven

### CODE REVIEW

**Comment by Sven-Seyfert** [comment]
### ✨[ReadMe preview](https://github.com/Sven-Seyfert/WebDriver/tree/readme-restructuring#readme)

**Comment by mlipok** [comment]
I think that: 
`Frequently Asked Questions` it does not have to be wrapped in its entirety, but only individual questions.

**Comment by mlipok** [comment]
`WinHttp_WebSocket UDF by @Danp2` should point to @Danp2 github account.

**Comment by Sven-Seyfert** [comment]
Hi @mlipok ,

good points 😀 .
- I fixed the missing new lines. 
- I added links to the project icons.

> I think that:
Frequently Asked Questions it does not have to be wrapped in its entirety, but only individual questions.

As far as I understood @Danp2 , it's at least what he wants. But let us discuss this further. Like I wrote above:
_"What I haven't done is move the FAQ section and also not stripping the content to a subset like you @Danp2 thought about the content of a FAQ."_

I mean maybe the whole section will be reworked, then it wouldn't make sense to change it now. But I am fine with any of your decisions 👍 .

> WinHttp_WebSocket UDF by @Danp2 should point to @Danp2 github account.
It's already done in "Preconditions" section. Do you really need it in the "Acknowledgements" section again?

**Comment by mlipok** [comment]
IMHO:
yours link to yours Youtube could be mentioned in Contributors, but should not be given at the start of this description.
https://youtube.com/channel/UCjPiWdl_h1CoYhZXaEC_AwA

**Comment by Sven-Seyfert** [comment]
@mlipok Oh damn, what a big fail!!! This is a copy and paste failure which shouldn't occure 🥴😂 . This wasn't intended, I jsut used that markdown link as a template 😅 .

**Comment by Sven-Seyfert** [comment]
> Michał: [...] could be mentioned in Contributors [...]

As far as I do not contribute with real valuable parts of the project like source code expanding or improvements, I see no need to mention me in the Acknowledgements area.

**Comment by mlipok** [comment]
> Hi @mlipok ,
> > I think that:
> > Frequently Asked Questions it does not have to be wrapped in its entirety, but only individual questions.
> 
> As far as I understood @Danp2 , it's at least what he wants. But let us discuss this further. Like I wrote above: _"What I haven't done is move the FAQ section and also not stripping the content to a subset like you @Danp2 thought about the content of a FAQ."_
> 
> I mean maybe the whole section will be reworked, then it wouldn't make sense to change it now. But I am fine with any of your decisions 👍 .
> 
> > WinHttp_WebSocket UDF by @Danp2 should point to @Danp2 github account.
> > It's already done in "Preconditions" section. Do you really need it in the "Acknowledgements" section again?

I was thinking that seeing this all 11 (as so far) questions at start is something that would be good.
image

I do not think that seeing all Q:A: but only titles of all FAQ.

**Comment by Sven-Seyfert** [comment]
@mlipok Ahh, okay I see. Thanks. Would be fine for me 😀 . Let's see what @Danp2 think about it.

**Comment by Sven-Seyfert** [comment]
FYI: I will add another commit regarding the project renaming "WebDriver" to "au3WebDriver" or "Au3WebDriver" 😉 .

**Comment by Danp2** [comment]
Personally, I'm not a fan of duplicating information across multiple sites. All this does is create a maintenance issue where it's likely that changes made to one site don't get replicated to other sites.

The FAQ section on the wiki covers many different scenarios, many of which don't belong in `readme.md` IMO.

**Comment by Sven-Seyfert** [comment]
I totally agree. We only need one source of truth which has to be maintained.
To achieve this, we should talk about which section should only be in the wiki and which one should only be in the ReadMe.

- Anyway, this could and should (IMHO) be a separate discussion.
- Or we agree to remove the whole FAQ section from the ReadMe (for now)?

**Comment by Danp2** [comment]
> I totally agree. We only need one source of truth which has to be maintained.

We can't prevent others from creating their own "documentation" on the internet, but we can make sure that what we provide on this repo is accurate and useful.

>  To achieve this, we should talk about which section should only be in the wiki and which one should only be in the ReadMe.

I think they each can have their own FAQ section, where each would cover different info.

> * Anyway, this could and should (IMHO) be a separate discussion.
> 
> * Or we agree to remove the whole FAQ section from the ReadMe (for now)?

I suggest that we do both for now.

**Comment by Sven-Seyfert** [comment]
> I suggest that we do both for now.

Accepted ✔

**Comment by mlipok** [comment]
> As a non native speaker like me it's sometimes hard to understand when "which", "that", "this" should be used.

Exactly like me.

> That's why I really appreciate comments like this Dan, thank you 😊 .

I join the thanks.

**Comment by Danp2** [comment]
> I believe that "good" should be "better" instead

@Sven-Seyfert Not sure what I was thinking when I typed that, but the correct word is "best". 😆 Apologies for leading you astray.

**Comment by Sven-Seyfert** [comment]
> Dan: Not sure what I was thinking when I typed that, [...]

- Could be me 😂 .
- Changed ✔

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,395 +1,840 @@
...
+
+---
+
+### Legend - Types of changes
+- `Added` for new features.
+- `Changed` for changes in existing functionality.
+- `Deprecated` for soon-to-be removed features.
+- `Removed` for now removed features.
+- `Fixed` for any bug fixes.
+- `Security` in case of vulnerabilities.
```
Comment (Danp2) [inline CHANGELOG.md:889]: New type for documentation or project changes?

**Pull request merged**